### PR TITLE
WIP: possible revision of building instructions for Mac

### PR DESCRIPTION
**Pull request by dsyme**
I was going through the build & getting started on OSX instructions in order to try to fix a bug I saw.

The first of the steps listed seemed unnecessary, I'm not sure about the other two either, so am working on this PR which I'll refine based on whether the steps are needed or not

I'll close it if it turns out all the steps are needed

### CODE REVIEW

File: developers/building-monodevelop.md [hunk 1]
```diff
@@ -104,10 +93,33 @@ Check out MD, configure using the Mac profile:
...
+
+   Comment: this did not appear necessary
+   
+2. Ensure that the Mono framework comes first in your PATH so you do not end up accidentally using conflicting versions of utilities from packages such as MacPorts:
+
+       export PATH="/Library/Frameworks/Mono.framework/Versions/Current/bin:$PATH"
+
+   Comment: this did not appear necessary
+
+3. Add the GTK+ libraries' location to the Mac dynamic loader path:
```
Comment (Therzok) [inline developers/building-monodevelop.md]: This should be done via the stub launcher on mac. We automatically add that to the path.

Comment (Therzok) [reply developers/building-monodevelop.md]: See https://github.com/mono/monodevelop/blob/master/main/build/MacOSX/monostub-utils.h#L219

Comment (dsyme) [reply developers/building-monodevelop.md]: @Therzok OK, I removed this

**Comment by dsyme** [comment]
I think this can be merged - I'll revist next time I build

**Pull request merged**